# HG changeset patch
# User andrew
# Date 1371235319 -3600
# Node ID 59e36a91e283804e0ff074e43ca67d4002113db9
# Parent  b1c99cf6c26d9df7ca7d02df1687064656c8ae71
8001032: Restrict object access
Summary: Restrict object access; fix reviewed also by Alexander Fomin
Reviewed-by: alanb, ahgross

diff --git a/test/Makefile b/test/Makefile
--- openjdk/jdk/test/Makefile
+++ openjdk/jdk/test/Makefile
@@ -483,7 +483,7 @@
 JDK_ALL_TARGETS += jdk_misc
 jdk_misc: demo javax/imageio javax/naming javax/print javax/script \
           javax/smartcardio javax/xml javax/sound com/sun/java com/sun/jndi \
-	  com/sun/org sun/misc sun/pisces
+	  com/sun/org com/sun/corba sun/misc sun/pisces
 	$(call RunSamevmBatch)
 
 # Stable samevm testruns (minus items from PROBLEM_LIST)
